client/components/mma/delivery/address/DeliveryAddressForm.tsx

```diff
@@ -433,6 +434,72 @@ export const DeliveryAddressUpdate = (props: WithProductType<ProductType>) => {
 		};
 	`;
 
+	const hasNationalDelivery = Object.values(
+		contactIdToArrayOfProductDetailAndProductType,
+	)
+		.flatMap(flattenEquivalent)
+		.some(({ productDetail }) => {
+			console.log(productDetail);
+			return GROUPED_PRODUCT_TYPES.subscriptions
+				.mapGroupedToSpecific(productDetail)
+				.productType.includes('nationaldelivery');
+		});
+
+	console.log(contactIdToArrayOfProductDetailAndProductType);
+
+	console.log(hasNationalDelivery);
+
+	if (hasNationalDelivery) {
+		return (
+			<>
+				<p
+					css={css`
+						display: block;
+						${textSans.medium()};
+						border: 4px solid ${palette.brand['500']};
+						padding: ${space[5]}px ${space[5]}px ${space[5]}px 49px;
+						margin-top: 12px;
+						position: relative;
+						${from.tablet} {
+							display: inline-block;
+							vertical-align: top;
+							width: calc(100% - (30ch + ${space[3]}px + 2px));
+						}
+					`}
+				>
+					<i
+						css={css`
+							width: 17px;
+							height: 17px;
+							position: absolute;
+							top: ${space[5]}px;
+							left: ${space[5]}px;
+						`}
+					>
+						<InfoIconDark fillColor={palette.brand[500]} />
+					</i>
+					Changed address? Please{' '}
+					<span
+						css={css`
+							cursor: pointer;
+							color: ${palette.brand[500]};
+							text-decoration: underline;
+						`}
+						onClick={() =>
+							setTopCallCentreNumbersVisibility(
+								!showTopCallCentreNumbers,
+							)
+						}
+					>
+						call our customer support team
+					</span>
```

Comment:
```suggestion
					</span>{' '}
```
looks like there's a missing space here in the screenshot